Cargo.toml

```diff
@@ -5,12 +5,18 @@ authors = ["Lonami Exo <totufals@hotmail.com>"]
 edition = "2018"
 
 [dependencies]
-grammers-client = { path = "../grammers/lib/grammers-client" }
-grammers-session = { path = "../grammers/lib/grammers-session" }
+grammers-client = { git = "https://github.com/Lonami/grammers.git", rev = "d91dc82" }
+grammers-session = { git = "https://github.com/Lonami/grammers.git", rev = "d91dc82" }
```

Comment:
I would prefer to revert this. Local paths can be up-to-date too. And development is easier. So I see no reason for this.